### DISCUSSION TITLE
Overheating issues

### DISCUSSION THREAD

**Discussion by rrlevy** [tesla-android/issue-tracker#49] [Q&A]
Since I created my first Tesla Android setup, I've been having issues with my Raspberry Pies overheating and still coundn't solve them.

I'm trying different cases in order to have a functioning system, and I wanted to check what are you guys doing as a solution.

At first, I atempted using a standard plastic case with heatsinks for the Pi and a Fan on the plastic case, for the Android Pi. For the Linux Pi, I used a [metal case](https://www.aliexpress.com/item/1005003327750007.html) with a fan provided by Geekworm (X630-C1) and the X630-A2 board that includes another fan. In total, I had 3 fans in the whole system, which I believed would be enough.

This was the complete setup:

![image](https://user-images.githubusercontent.com/14061585/172725015-1cd85b77-71d1-4644-94c1-fb48c9ac70df.png)

However, I was having a lot of overheating issues, and I decided to go for a more "open" approach, so that the Raspis could have better ventilation:

![image](https://user-images.githubusercontent.com/14061585/172725715-3b0ef45a-ae32-4b4f-b3fe-77b26c0d8610.png)

This alternative had only the fan included in the X630-A2 board, and the heatsink for the Android Pi.

This worked a little bit better, but I'm still having issues.

Here is a video with an example of what happens when the system overheats:

https://www.youtube.com/watch?v=ZZ4PsGihec8

If I open my armrest and let the heat out, the video issue is fixed immediately.

Another frequent problem I have is that the screen gets totally "blank" and I need a browser reload for it to work. 

My next atempt will be to use something like this with more fans and an open stack:

![image](https://user-images.githubusercontent.com/14061585/172726210-2d12c256-73a2-4d88-992e-a8fb098cc966.png)

I would love to hear how you guys are dealing with the heating issues!

**Comment by mbhforum** [comment]
Unfortunately, I don't have a solution for you. I will say I've had 3 SD cards go bad already and I wonder if the heat is playing a role in this.  I live in Florida and the car can obviously get very hot when parked.  Not sure this is sustainable long term.

**Reply by rrlevy** [reply]
The problem is that the system is heating up even during the winter here in Brazil! The heat builds up mostly because of the Raspis and not due to external climate. The problem usually happens when I'm driving and the system is being used.
